pom.xml

```diff
@@ -19,8 +19,8 @@
   <properties>
     <java.version>1.8</java.version>
     <avro.version>1.8.0</avro.version>
-    <!-- informative only, used in about template substitution -->
-    <scala.version>2.10</scala.version>
+    <scala.version>2.10.6</scala.version>
+    <scala.version.prefix>2.10</scala.version.prefix>
```

Comment:
changing this also requires a change to https://github.com/bigdatagenomics/adam/blob/master/adam-cli/src/main/java-templates/org/bdgenomics/adam/cli/About.java#L28